test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -322,24 +326,25 @@ u256 EVMInstructionInterpreter::eval(
 		return (0xdddddd + arg[1]) & u256("0xffffffffffffffffffffffffffffffffffffffff");
 	case Instruction::CALL:
 	case Instruction::CALLCODE:
-		// TODO assign returndata
 		accessMemory(arg[3], arg[4]);
 		accessMemory(arg[5], arg[6]);
+		m_state.calldata = readMemory(arg[3], arg[4]);
```

Comment:
Could we do this only if address is equal to self? Otherwise, we are simply populating call data without actually executing the call, and may be this gives the impression that the call took place although it actually didn't?